xml/MAIN-SBP-SLSA.xml

```diff
@@ -0,0 +1,1515 @@
+   </para>
+   <table>
+    <title>SLSA4 Build (process) requirements and SUSE's OBS</title>
+    <tgroup cols="3">
+     <colspec colname="c1" align="left"/>
+     <colspec colname="c2" align="center"/>
+     <colspec colname="c3" align="center"/>
+     <thead>
+      <row>
+       <entry>
+       Requirement
+      </entry>
+       <entry>
+       Required at SLSA4
+      </entry>
+       <entry>
+       Achieved
+      </entry>
+      </row>
+     </thead>
+     <tbody>
+      <row>
+       <entry>
+       Scripted build
+      </entry>
+       <entry>
+       &#x2713;
+      </entry>
+       <entry>
+       &#x2713;
+      </entry>
+      </row>
+      <row>
+       <entry>
+       Build Service
+      </entry>
+       <entry>
+       &#x2713;
+      </entry>
+       <entry>
+       &#x2713;
+      </entry>
+      </row>
+      <row>
+       <entry>
+       Build as code
+      </entry>
+       <entry>
+       &#x2713;
+      </entry>
+       <entry>
+       &#x2713;
+      </entry>
+      </row>
+      <row>
+       <entry>
+       Ephemeral environment
+      </entry>
+       <entry>
+       &#x2713;
+      </entry>
+       <entry>
+       &#x2713;
+      </entry>
+      </row>
+      <row>
+       <entry>
+       Isolated
+      </entry>
+       <entry>
+       &#x2713;
+      </entry>
+       <entry>
+       &#x2713;
+      </entry>
+      </row>
+      <row>
+       <entry>
+       Parameterless
+      </entry>
+       <entry>
+       &#x2713;
+      </entry>
+       <entry>
+       &#x2713;
+      </entry>
+      </row>
+      <row>
+       <entry>
+       Hermetic
+      </entry>
+       <entry>
+       &#x2713;
+      </entry>
+       <entry>
+       &#x2713;
+      </entry>
+      </row>
+      <row>
+       <entry>
+       Reproducible
+      </entry>
+       <entry>
+       &#x25cb;
+      </entry>
+       <entry>
+       &#x274c;
+      </entry>
+      </row>
+     </tbody>
+    </tgroup>
+   </table>
+   <para>
+    See how SUSE's current software building pipeline with the Open Build
+    Service (OBS) meets the above requirements:
+   </para>
+   <variablelist>
+    <varlistentry>
+     <term>Scripted build</term>
+     <listitem>
+      <itemizedlist>
+       <listitem>
+        <formalpara>
+         <title>SLSA L4 requirement</title>
+         <para>
+          Define all build steps in any kind of build script. Manual
+          interaction is limited to invoking the build script itself.
+         </para>
+        </formalpara>
+       </listitem>
+       <listitem>
+        <formalpara>
+         <title>SUSE's processes</title>
+         <para>
+          This is achieved by the SUSE build script also used by the OBS. Even
+          the decision to invoke a build is made by OBS based on the submitted
+          code changes or other builds.
+         </para>
+        </formalpara>
+       </listitem>
+      </itemizedlist>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>Build service</term>
```

Comment:
This does not refer to the Open Build Service by SUSE. In that case, we would have used it as a name and used the "Service". This refers to the original requirement by Google which asks for a build service of any kind and this would not require for an upppercase S, or would it?